### PR TITLE
Tests: added more reducers test cases

### PR DESCRIPTION
**Pull request by LikerFeed**
Added more test cases for clear, clearAll reducers.

### CODE REVIEW

File: src/redux/slices/home/reducers.test.ts
Comment (Dimdim28) [inline src/redux/slices/home/reducers.test.ts:1]: Cool, you handled cass with different last page and totalpages, thanks =)

**Review by Dimdim28 — APPROVED** [review]
Cool, now please add tests to yuur extra reducers =)

File: src/pages/Home/FiltersBar/Categories/Category/Category.test.tsx
Comment (crazysparrow69) [inline src/pages/Home/FiltersBar/Categories/Category/Category.test.tsx:1]: U covered all possible situations, impressive

**Review by crazysparrow69 — APPROVED** [review]
Nice job

**Pull request merged**